sql/changes/1.6/constrain_cr_report_line_entries.sql

```diff
@@ -0,0 +1 @@
+CREATE UNIQUE INDEX cr_report_line_entry_idx ON cr_report_line(report_id,ledger_id);
```

Comment:
Maybe we need 

```sql
CREATE UNIQUE INDEX cr_report_line_entry_idx ON cr_report_line(report_id,ledger_id) WHERE clear_date NOT NULL;
```

So that reports can be saved, with lines included in multiple reports. Just can't accept/post two reports which include the same line.

Reply:
Why should we ever want to allow duplication of a transaction in reconciliation reports? That is one of the source of the problems in reconciliations.

Reply:
Well, maybe I don't have `Delete` rights, but I need to prepare the recon report for my manager, so after submitting one, I need to start a new one because my manager isn't available to delete the first report until the next day at which time he'll be deleting one and accepting the other?